tests/plugins/test_pulp.py

```diff
@@ -45,7 +45,7 @@ class X(object):
     base_image = ImageName(repo="qwe", tag="asd")
 
 
-def prepare(check_repo_retval=0, existing_layers=[],
```

Comment:
IMHO this changes behavior. `None` and `[]` have different meaning here.

You can use

```
_undefined = object()

def prepare(.., existing_layers=_undefined, ...):
    if existing_layers is _undefined:
       existing_layers = []
```

another way is, that all tests must be updated accordingly (preferred given the small number of tests using this function)

Reply:
Ok, updated tests